ethstats/state.go

```diff
@@ -214,6 +214,28 @@ func (s *State) WriteNodeStats(nodeId string, stats *NodeStats) error {
 	return nil
 }
 
+func (s *State) DeleteOlderData(days int) error {
```

Comment:
This is not tested

Reply:
Tested manually. Automatic testing is not possible since it's 'created_at' is set by postgres.

Reply:
Please refactor this to unit test it. Instead of doing `DeleteOlderData(days int)` you can write `DeleteOlderData(until time.Time)`. This way, you could write a unit test that do not depend on Postgresql.